hoodie-common/src/main/java/com/uber/hoodie/common/table/TableFileSystemView.java

```diff
@@ -30,9 +30,9 @@
 public interface TableFileSystemView {
 
   /**
-   * ReadOptimizedView - methods to provide a view of columnar data files only.
+   * ReadOptimizedView with methods to only access latest version of file for the instant(s) passed.
    */
-  interface ReadOptimizedView {
+  interface LatestVersionOnlyReadOptimizedView {
```

Comment:
Limiting the API usable by LatestFileSliceOnlyFSView

Reply:
rename to `ReadOptimizedViewWithLatestSlice`?